src/stories/PopoverMenu.stories.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PopoverMenu from '../components/PopoverMenu';
+import * as Expensicons from '../components/Icon/Expensicons';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+import MenuItem from '../components/MenuItem';
+import colors from '../styles/colors';
+
+/**
+ * We use the Component Story Format for writing stories. Follow the docs here:
+ *
+ * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
+ */
+const story = {
+    title: 'Components/PopoverMenu',
+    component: PopoverMenu,
+};
+
+// eslint-disable-next-line react/jsx-props-no-spreading
+const Template = (args) => {
+    const [isVisible, setIsVisible] = React.useState(false);
+    const toggleVisibility = () => setIsVisible(!isVisible);
+    return (
+        <>
+            <MenuItem {...args.menuItem} onPress={toggleVisibility} wrapperStyle={isVisible ? [{backgroundColor: colors.gray2}] : []} />
+            <SafeAreaProvider>
```

Comment:
Wrapped `PopoverMenu` with `SafeAreaProvider` instead of wrapping `BasePopoverMenu` as it was breaking the ui.